### PR TITLE
feat: updated setting page style and contents

### PR DESCRIPTION
**Pull request by sugakou44**
# Overview
- [x] Updated setting page layout and style
- [x] Added Cookie Banner activation instruction

# Preview

image

### CODE REVIEW

**Review by huangsri — APPROVED** [review]

File: settings.php [hunk 1]
```diff
@@ -6,13 +6,82 @@
...
+                (Or sign up
+                    <a
+                        href="https://app.cookiewow.com/users/sign_up"
+                        target="_blank"
+                        rel="norefferer"
+                    >here</a>
+                )
+            </h3>
+            <ol>
+                <li>Add your wordpress domain</li>
```
Comment (nimid) [inline settings.php]: @sugakou44 Typo.

```suggestion
                <li>Add your WordPress domain</li>
```

File: settings.php [hunk 2]
```diff
@@ -6,13 +6,82 @@
...
+                <li>Go to your domain dashboard and click Customize</li>
+                <li>Copy Banner ID from “Get script” button</li>
+                <li>Paste Banner ID on Wordpress plug-in and click “Save Changes”</li>
```
Comment (nimid) [inline settings.php]: @sugakou44 Typo.

```suggestion
                <li>Paste Banner ID on WordPress plug-in and click “Save Changes”</li>
```

File: settings.php (hunk 1 above)
Comment (sugakou44) [reply settings.php]: fixed in https://github.com/datawowio/cookiewow-wordpress/pull/8/commits/e96287f433845cc3414605847b15de70478351b5

File: settings.php (hunk 2 above)
Comment (sugakou44) [reply settings.php]: fixed in https://github.com/datawowio/cookiewow-wordpress/pull/8/commits/e96287f433845cc3414605847b15de70478351b5

**Review by nimid — APPROVED** [review]

**Pull request merged**